extensions/wikia/CuratedContent/crowdin.conf

```diff
@@ -0,0 +1,6 @@
+[crowdin.project]
+projectid = "ext-curated-content"
+format = "mediawiki"
+repo.breanch = "${projectid}-translations"
```

Comment:
It is the default branch name to checkout and commit translations to if you use the commit (and optionally push) flags without passing a branch name with `-b`. By default it will just use `${projectid}-crowdin` so you don't need to set it necessarily, and most likely people will pass a ticket number i.e. `crowdin download -c -p -b MAIN-1234`. This was based on the expected workflow, but we can change the default behaviour as needed based on the feedback of teams using the tools.